src/dropdown/dropdown.component.ts

```diff
@@ -407,7 +409,7 @@ export class Dropdown implements OnInit, AfterContentInit, AfterViewInit, OnDest
 		protected elementRef: ElementRef,
 		protected i18n: I18n,
 		protected dropdownService: DropdownService,
-		protected elementService: ElementService) {}
+		protected elementService: ElementService) { }
```

Comment:
With the above changes, we would also have to inject changeDetection. Also requires some linting here:

```suggestion
		protected elementService: ElementService,
		private changeDetectionRef: ChangeDetectionRef
	) {}
```